include/battle.h

```diff
@@ -234,7 +234,11 @@ struct AI_ThinkingStruct /* 0x2016800 */
 
 struct BattleStruct /* 0x2000000 */
 {
-    u8 filler0[0x15DDE];
+    /*0x00000*/ u8 unk0;
```

Comment:
This was part of an early attempt to merge ewram structs. This can be ignored as I don't know if its actually part of this.